src/User.js

```diff
@@ -0,0 +1,7 @@
+class User {
+  constructor() {
+
+  }
```

Comment:
What properties were you thinking of adding?

Reply:
* The User class will have an id, name, username and password
* These will be inherited by the Traveler and Agent classes